calico-vpp-agent/watchers/uplink_route_watcher.go

```diff
@@ -34,6 +43,22 @@ type RouteWatcher struct {
 	stop              bool
 	lock              sync.Mutex
```

Comment:
`RouteWatcher.lock` can probably go, we seem to be now only using the resources it protects in a single loop